features/steps/solvers.py

```diff
@@ -47,8 +40,48 @@ def step_impl(context):
 @then("they should include at least the minimum set of solvers")
 def step_impl(context):
     """Verify all requested solvers are available."""
-    print(context.result)
-    requested_solvers = context.result["requested_solvers"]
     available_solvers = context.result["available_solvers"]
-    for r_s in requested_solvers:
-        assert_that(available_solvers, has_item(r_s))
+    for a_s in available_solvers:
+        assert re.match("solver-(rhel|fedora)-.*\d-py3\d", a_s)  # noqa: W605
+
+
+@then("schedule solver analyses for package {package_name} with version {package_version}")
+def step_impl(context, package_name, package_version):
+    """Schedule solver jobs for all available solvers."""
+    url = f"{context.scheme}://{context.management_api_host}/api/v1/solver/python"
+    response = requests.post(
+        url,
+        json={"package_name": package_name, "version_specifier": "==" + package_version},
+        params={"secret": os.environ["THOTH_MANAGEMENT_API_SECRET"]},
```

Comment:
@fridex @pacospace Is it OK to get the value for `THOTH_MANAGEMENT_API_SECRET` here or should I move somewhere else?

Reply:
You can move it to the top of this file.